portfolio/src/main/webapp/script.js

```diff
@@ -142,41 +147,66 @@ function getMessage() {
 function handleResponse(response) {
   const textPromise = response.text();
   
-  textPromise.then(addSingleMessageToDom);
+  textPromise.then(addSingleCommentToDom);
 }
 
 /**
- * Adds a single message {@code message} inside of the div
+ * Adds a single {@code comment} inside of the div
  * message-container. 
  */
-function addSingleMessageToDom(message) {
-  const messageContainer = document.getElementById('message-container');
-  messageContainer.innerHTML = message;
+function addSingleCommentToDom(comment) {
+  const commentContainer = document.getElementById(COMMENT_CONTAINER);
+  commentContainer.innerHTML = message;
 }
 
 /**
- * Adds {@code messages} to the DOM as list elements.
+ * Adds {@code comments} to the DOM as list elements.
  */
-function addMultipleMessagesToDom(messages) {
-  const messageContainer = document.getElementById('message-container');
-  const ulElement = document.createElement('ul');
-  messageContainer.appendChild(ulElement);
+function addMultipleMessagesToDom(comments) {
+  const commentContainer = document.getElementById(COMMENT_CONTAINER);
 
-  for (let key in messages) {
-    appendTextToList(messages[key], ulElement);
+  // Removes the ul tag in the container if there is one to prevent having
+  // multiple sets of ul tags every time the number of comments is changed.
+  if (commentContainer.firstChild) {
+    commentContainer.removeChild(commentContainer.firstChild);
   }
+  const ulElement = document.createElement('ul');
+  commentContainer.appendChild(ulElement);
+
+  comments.forEach((comment) => {
+    appendTextToList(comment.text, ulElement);
+  });
 }
 
 /**
- * Fetches the message from the JSON server /data and adds it to the DOM.
- *
- * <p>Method is called everytime the page is refreshed.
+ * Fetches the comment from the JSON server /data and adds it to the DOM.
+ * 
+ * <p>{@code pageReloadBoolean} indicates whether the method is called
+ * when the page refreshes or when a new comment amount is inputted.
  */
-function getMessageFromJSON() {
-  fetch('/data')
+function getMessageFromJSON(pageReloadBoolean) {
+  let selectedIndex;
+  let amountSelector = document.getElementById(COMMENT_AMOUNT);
+  
+  // Retrieves the selected index from local storage if there is a value for it
+  // because after a page reload, the selected index is set back to its default
+  // value of 0. 
```

Comment:
This sentence seems quite long. Can you split it. I think it would add clarity.

Reply:
Done!